bench/irmin-pack/tree.ml

```diff
@@ -74,8 +201,9 @@ module Parse_trace = struct
     | hd :: tl -> hd :: flatten_key_suffix tl
     | [] -> []
 
-  (** This function flattens all the 6 step-long chunks forming 38 byte-long
-      hashes to a single step.
+  (** This function flattens all the 6 step-long chunks forming 40 byte-long
+      hashes to a single step. Those flattenings are performed during the trace
+      replay, i.e. they count in the total time.
```

Comment:
isn't this a problem if we want to compare flatten vs not-flatten?

Reply:
It is, if we look at the total time spent in the bench, but it is not if we look at the sum of the time spent in all operations.
![image](https://user-images.githubusercontent.com/9285880/110652494-b7a7e300-81bc-11eb-99af-4d7f4629412e.png)